static/sass/_pattern_cube-animation.scss

```diff
@@ -0,0 +1,70 @@
+$size: (
+  "width": 169,
+  "height": 195,
+);
+$speed: 2s;
+$positions: (
+  "1": (
+    "x": 100,
+    "y": 1,
+    "to": "6",
+  ),
+  "2": (
+    "x": 183,
+    "y": 49,
+    "to": "1",
+  ),
+  "3": (
+    "x": 183,
+    "y": 145.5,
+    "to": "4",
+  ),
+  "4": (
+    "x": 100,
+    "y": 193,
+    "to": "5",
+  ),
+  "5": (
+    "x": 16,
+    "y": 145.5,
+    "to": "2",
+  ),
+  "6": (
+    "x": 16,
+    "y": 49,
+    "to": "3",
+  ),
+);
+
+.cube {
```

Comment:
Can you wrap all these styles in the container class. At the moment that would be `.hero-animation`.

Reply:
Otherwise, we have a global class called `cube`. Which could be used somewhere else.

Reply:
If you update the class below to `.p-cube-animaiton` then this should be BEMed like this `.p-cube-animaiton__cube`